static/sass/_patterns_fluid-grid.scss

```diff
@@ -16,6 +16,8 @@
     justify-content: start;
 
     .p-fluid-grid__item {
+      width: 50%;
```

Comment:
Why adding this, what does that change?

Reply:
This changes the smallest size so it's still col-2, rather than 1-col. Even on mobile it looks better in 2 cols.